akka-stream/src/main/scala/akka/stream/javadsl/Queue.scala

```diff
@@ -63,3 +63,14 @@ trait SinkQueue[T] {
    */
   def pull(): CompletionStage[Optional[T]]
 }
+
+/**
+ * This trait adds cancel support to [[SinkQueue]].
+ */
```

Comment:
Yeah, I meant its here for outside usage, but not for subtyping, `class MyClass extends SinkQueueWithCancel[Any]`.  Not super important though.